lightning/src/ln/msgs.rs

```diff
@@ -1503,10 +1529,38 @@ impl Readable for ErrorMessage {
 		Ok(Self {
 			channel_id: Readable::read(r)?,
 			data: {
-				let mut sz: usize = <u16 as Readable>::read(r)? as usize;
-				let data = read_to_end(r)?;
-				sz = cmp::min(data.len(), sz);
```

Comment:
Looks like dropping this introduced a compiler warning for unused import.

Reply:
Oops, right, it was in the last commit so went ahead and ninja'd an edit.